homeassistant/components/rfxtrx/__init__.py

```diff
@@ -204,7 +206,7 @@ def async_handle_receive(event):
             pt2262_devices.append(event.device.id_string)
 
         device_entry = device_registry.async_get_device(
-            identifiers={(DOMAIN, *device_id)},
+            identifiers={(DOMAIN, *device_id)},  # type: ignore[arg-type]
```

Comment:
Device id appears to be a NamedTuple, but my impression is that device identifiers are typically a a (deviceid, id) pair but this could have more than two values.  I assume that's why the type ignore is here.  Is this an existing bug or does this code  work and just need typing help?

Reply:
It works. Its legacy before this was standardized. I have a draft for changing this here: https://github.com/home-assistant/core/pull/75119